src/issue-details/issue-details-slice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
This is a core change: this defines all the redux handling for the `issueDetails` part of the redux state